toolbox/gpu-operator/diagnose.sh

```diff
@@ -73,7 +73,7 @@ do_test "The cluster is reachable" \
 
 do_test "The cluster has the NFD operator" \
         "There is no NFD-labelled node. Is the NFD operator running properly?" \
-        toolbox/nfd/has_nfd_labels.sh
+        toolbox/nfd/has_labels.sh
```

Comment:
should be `./run.py nfd has_labels` I guess

Reply:
and the whole `diagnose.sh` script should be updated,
I guess the testing didn't fail because `toolbox/nfd/has_labels.sh` failure was treated as if no NFD label were found